e2e-test/trace/build.gradle

```diff
@@ -148,120 +154,104 @@ task stopOpenSearchDockerContainer(type: DockerStopContainer) {
-            "serviceMapDataPrepper1", "dataprepper1", 21890, 4900, "/app/${SERVICE_MAP_PIPELINE_YAML}")
-    def createDataPrepper2Task = createDataPrepperDockerContainer(
-            "serviceMapDataPrepper2", "dataprepper2", 21891, 4901, "/app/${SERVICE_MAP_PIPELINE_YAML}")
-    def startDataPrepper1Task = startDataPrepperDockerContainer(createDataPrepper1Task as DockerCreateContainer)
-    def startDataPrepper2Task = startDataPrepperDockerContainer(createDataPrepper2Task as DockerCreateContainer)
-    dependsOn startDataPrepper1Task
-    dependsOn startDataPrepper2Task
-    startDataPrepper1Task.mustRunAfter 'startOpenSearchDockerContainer'
-    startDataPrepper2Task.mustRunAfter 'startOpenSearchDockerContainer'
-    // wait for data-preppers to be ready
-    doFirst {
-        sleep(10*1000)
-    }
+def rawSpanDataPrepperEventFromBuild = createDataPrepperDockerContainer(
+        "rawSpanDataPrepperEventFromBuild", "dataprepper1", 21890, 4900, "/app/${RAW_SPAN_PIPELINE_PEER_FORWARDER_EVENT_TYPE_YAML}")
+def rawSpanDataPrepperLatestFromPull = createDataPrepperDockerContainerFromPullImage(
+        "rawSpanDataPrepperLatestFromPull", "dataprepper2", 21891, 4901, "src/integrationTest/resources/${RAW_SPAN_PIPELINE_LATEST_RELEASE_YAML}")
 
-    description = 'Runs the service-map integration tests.'
-    group = 'verification'
-    testClassesDirs = sourceSets.integrationTest.output.classesDirs
-    classpath = sourceSets.integrationTest.runtimeClasspath
+def rawSpanEventLatestReleaseCompatibilityEndToEndTest = createEndToEndTest("rawSpanEventLatestReleaseCompatibilityEndToEndTest",
+        includeRawSpanTestsMatchPattern,
+        rawSpanDataPrepperEventFromBuild, rawSpanDataPrepperLatestFromPull)
 
-    filter {
-        includeTestsMatching "com.amazon.dataprepper.integration.trace.EndToEndServiceMapTest*"
-    }
+def includeServiceMapTestsMatchPattern = "com.amazon.dataprepper.integration.trace.EndToEndServiceMapTest*"
 
-    finalizedBy stopOpenSearchDockerContainer
-    def stopDataPrepper1Task = stopDataPrepperDockerContainer(startDataPrepper1Task as DockerStartContainer)
-    def stopDataPrepper2Task = stopDataPrepperDockerContainer(startDataPrepper2Task as DockerStartContainer)
-    finalizedBy stopDataPrepper1Task
-    finalizedBy stopDataPrepper2Task
-    finalizedBy removeDataPrepperDockerContainer(stopDataPrepper1Task as DockerStopContainer)
-    finalizedBy removeDataPrepperDockerContainer(stopDataPrepper2Task as DockerStopContainer)
-    finalizedBy removeDataPrepperNetwork
-}
+def createServiceMapDataPrepperOTLP1Task = createDataPrepperDockerContainer(
+        "serviceMapDataPrepper1", "dataprepper1", 21890, 4900, "/app/${SERVICE_MAP_PIPELINE_YAML}")
+def createServiceMapDataPrepperOTLP2Task = createDataPrepperDockerContainer(
+        "serviceMapDataPrepper2", "dataprepper2", 21891, 4901, "/app/${SERVICE_MAP_PIPELINE_YAML}")
+
+// TODO: replace with Event type in 2.0
```

Comment:
Let's provide a link to a Github issue and include a reference to this work in that issue. This way we can easily track the work required when we pick it up.